docs/content/en/docs/configuration/keys.md

```diff
@@ -618,9 +618,9 @@ See also:
 
 Configure if HAProxy should maintain client requests to the same backend server.
 
-* `affinity`: the only supported option is `cookie`. If declared, clients will receive a cookie with a hash of the server it should be fidelized to.
+* `affinity`: the only supported option is `cookie`. If declared, clients will receive a cookie with a hash of the server to which it should be linked.
```

Comment:
I [asked about this](https://kubernetes.slack.com/archives/C01D8GKKT5H/p1763413043210239).

Other possible phrasings would involve `delivered`, `terminated by`, or `persisted`. The original form also ended with a preposition which also made understanding the sentence hard -- even my fix isn't great wrt that.

It'd be better if the sentence explained that the ingress would decode the cookie and use it to return traffic to the hashed server (when available). And it'd be nice if it explained what happens if that server is [410 gone](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/410).